react/features/base/media/components/web/VideoTrack.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import AbstractVideoTrack from '../AbstractVideoTrack';
+
+/**
+ * Component that renders a video element for a passed in video track.
+ *
+ * @extends AbstractVideoTrack
+ */
+class VideoTrack extends AbstractVideoTrack {
+    /**
+     * Default values for {@code VideoTrack} component's properties.
+     *
+     * @static
+     */
+    static defaultProps = {
```

Comment:
They are used but probably aren't needed anymore. They just make sure className and id aren't undefined but I'm sure react knows how to handle that.